webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -79,15 +106,28 @@ def update_correlation_plot(
                 continue
             valid_responses = []
             for response in responses:
-                response_df = ensemble.responses[response].data_df()
-                if response_df.empty:
+                response_el = ensemble.responses[response]
+                if response_el.empty:
```

Comment:
What does `el` refer to in `response_el`?

Reply:
`el` stands for element but I will refactor to replace response_el with response and response from ln 109 with response_name to be more specific